### DISCUSSION TITLE
Frame Tool Does Not Work on My Longer 5

### DISCUSSION THREAD

**Discussion by martdona1217** [arkypita/LaserGRBL#2154] [Q&A]
When I try to frame a design to ensure the workspace is right the laser does not move or it moves in an area that is only 5mm or so.  I get the error message "Bad Number Format" and  three lines of "Undefined Feed Rate" .  I am running Windows 11.   The feature works fine on my friends Windows 10 computer.  I have uninstalled and reinstalled the program more than once.  I have downloaded the drivers I can find in the program.  I am not sure what else I need to do.  Any help or guidance would be greatly appreciated.  I am new to this and not sure where else to research.  :-)

**Comment by Tribo-2dot0** [comment]
Right click on the Frame button, Edit Button... Replace the GCode with this code:

G0 X[left] Y[bottom]
M3 S5 F1500
G1 Y[top] 
G1 X[right]
G1 Y[bottom] 
G1 X[left]
M5
G0 X0 Y0
